src/jmc/compile/command/builtin_function/jmc_command.py

```diff
@@ -623,136 +622,52 @@ class AdvancementRevoke(JMCFunction):
     def call(self) -> str:
         advancement = self.args["advancement"]
         target = self.args["target"]
-        selection = self.args["selection"]
+        type = self.args["type"]
```

Comment:
`type` is overriding built-in `type` of python
name should be changed to `type_` or something else